Index: cinnamon-session/capplet/csm-properties-dialog.c
===================================================================
--- cinnamon-session.orig/capplet/csm-properties-dialog.c	2014-10-06 13:05:29.488379725 +0200
+++ cinnamon-session/capplet/csm-properties-dialog.c	2014-10-06 13:05:29.488379725 +0200
@@ -678,7 +678,9 @@
 
         setup_dialog (dialog);
 
+#if !GTK_CHECK_VERSION (3, 14, 0)
         gtk_widget_show (GTK_WIDGET (dialog));
+#endif
 
         return G_OBJECT (dialog);
 }
Index: cinnamon-session/capplet/csm-app-dialog.c
===================================================================
--- cinnamon-session.orig/capplet/csm-app-dialog.c	2014-10-06 13:05:29.488379725 +0200
+++ cinnamon-session/capplet/csm-app-dialog.c	2014-10-06 13:05:29.488379725 +0200
@@ -270,7 +270,9 @@
 
         setup_dialog (dialog);
 
+#if !GTK_CHECK_VERSION (3, 14, 0)
         gtk_widget_show_all (GTK_WIDGET (dialog));
+#endif
 
         return G_OBJECT (dialog);
 }
